<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuadListDialog</class>
 <widget class="QDialog" name="QuadListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quad-Structure Scan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc/icons.qrc">
    <normaloff>:/icons/quad.png</normaloff>:/icons/quad.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Quad-Structures</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QTableWidget" name="listQuadStruct">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Structure</string>
         </property>
         <property name="toolTip">
          <string>type of quad-structure</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Distance</string>
         </property>
         <property name="toolTip">
          <string>distance to origin</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Optimal centre</string>
         </property>
         <property name="toolTip">
          <string>optimal AFK location</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Enclosing range</string>
         </property>
         <property name="toolTip">
          <string>distance to furthest spawning space from optimal AFK position</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Spawn area</string>
         </property>
         <property name="toolTip">
          <string>horizontal bounding box area inside a 128 block range</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelMsg">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridMCSeed">
     <item row="0" column="2">
      <widget class="QLabel" name="labelSeed">
       <property name="toolTip">
        <string>Seed can be an integer or text. Leave empty for a random seed.</string>
       </property>
       <property name="text">
        <string>seed:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineSeed">
       <property name="toolTip">
        <string>press enter to accept</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labeMC">
       <property name="toolTip">
        <string>Minecraft version</string>
       </property>
       <property name="text">
        <string>MC</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="buttonGo">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxMC">
       <property name="toolTip">
        <string>Minecraft version</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
       <property name="minimumContentsLength">
        <number>6</number>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>16</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>1.18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.0</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="buttonClose">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset theme="window-close">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
